crates/chia-sdk-driver/src/puzzle.rs

```diff
@@ -97,3 +99,42 @@ pub struct RawPuzzle {
     pub puzzle_hash: TreeHash,
     pub ptr: NodePtr,
 }
+
+impl<E> ToClvm<E> for Puzzle
```

Comment:
another ToClvm

Reply:
This was changed while you were looking at it

Reply:
![image](https://github.com/user-attachments/assets/5ff4ae43-14ae-4094-82ba-235ece86bbdb)